src/vs/workbench/test/workbenchTestServices.ts

```diff
@@ -738,6 +738,18 @@ export class TestFileService implements IFileService {
 		});
 	}
 
+	resolveStringStream(resource: URI, options?: IResolveContentOptions): IStringStream {
+		const callback: { [name: string]: Function } = Object.create(null);
```

Comment:
For this case we can put `{[name: string]: Function}` to `custom-typings.d.ts`

```typescript
interface Dictionary<T extends any> {
    [key: string]: T;
}
```

Other developers will thank you for that.